AllAboutGraphQL/Blog.MD

```diff
@@ -1,55 +1,47 @@
 
-## Myth You can't have REST endpoints in your implementation with GraphQL. 
-You can plugin multiple REST endpoints or even multiple GraphQL endpoints in your application. Although it is not a best practice to have multiple REST endpoints, it is technically possible. 
+## Myth: GraphQL is a database language. 
+GraphQL is a programming language - specifically a query language. GraphQL's spec defines how GraphQL runtimes should implement the language and how data should be communicated between client and server.
 
-## Myth GraphQL is meant for green field project and it is difficult to introduce in an existing project. 
-GraphQL can be plugged in an existing project. You can start with one component of a business logic, plug in a GraphQL endpoint, and start fetching data through GraphQL. You don't need to scrap an entire project to start using GraphQL. If switching to GraphQL endpoint is still a daunting task, you can start by masking a REST endpoint into a GraphQL endpoint using resolvers. 
+## Myth: You can't have REST endpoints in your implementation with GraphQL. 
+You can plug in multiple REST endpoints or even multiple GraphQL endpoints in your application. Although it is not a best practice to have multiple REST endpoints, it is technically possible.
 
-## Myth GraphQL is only for frontend folks
-Although frontend folks like myself are really passionate about GraphQL, GraphQL is platform agnostic. In my opinion, the beauty of GraphQL's benefits starts from the inside out - backend to frontend. 
+## Myth: GraphQL is difficult to introduce in an existing project. 
+GraphQL can be plugged into an existing project. You can start with one component of business logic, plug in a GraphQL endpoint, and start fetching data through GraphQL. You don't need to scrap an entire project to start using GraphQL. If switching to GraphQL endpoint is still a daunting task, you can start by masking a REST endpoint into a GraphQL endpoint using resolvers. 
 
+## Myth: GraphQL is only for front-end developers
+Although front-end folks like myself are really passionate about GraphQL, GraphQL is platform agnostic. In my opinion, the beauty of GraphQL's benefits starts from the inside out - backend to frontend.
 
-## Myth GraphQL will write database queries itself, I just need to specify schemas and the relation between them
-You will need to write the database queries depending on which GraphQL library you are using. Everything including resolvers, queries, mutations needs to be defined. 
\ No newline at end of file
+## Myth: GraphQL will write database queries itself, I just need to specify schemas and the relation between them
```

Comment:
There are a few libraries that will write database queries for you - [This one](https://github.com/neo4j-graphql/neo4j-graphql-js) is for Neo4j, and I _think_ Prisma does something like this too

Reply:
Thanks Harland. I wanted to communicate that certain libraries will do it for you, but it will make more sense to give examples of Neo4j and Prisma like you mentioned.